components/shared/webrender/rendering_context.rs

```diff
@@ -189,31 +155,32 @@ impl RenderingContext {
         device.bind_surface_to_context(context, surface).unwrap();
     }
 
-    pub fn device(&self) -> std::cell::Ref<Device> {
-        self.0.device.borrow()
-    }
-
+    #[allow(dead_code)]
     pub fn connection(&self) -> Connection {
         let device = &self.0.device.borrow();
         device.connection()
     }
 
+    #[allow(dead_code)]
```

Comment:
What is the reasoning behind these new `dead_code` annotations?

Reply:
They are probably not required now that `SurfmanRenderingContext` will remain in webrender crate, it's a byproduct from moving into servoshell. I'll do some reworking and will turn this PR into a draft state for now.